library/Mockery/Adapter/Phpunit/MockeryTestCase.php

```diff
@@ -37,4 +37,22 @@ protected function mockeryTestSetUp()
     protected function mockeryTestTearDown()
     {
     }
+
+    public function expectExceptionMessageRegEx($regularExpression)
```

Comment:
Would it have been better to leave the method name as `expectExceptionMessageRegExp` here? Or was there a clash in the signatures over the course of different PHPUnit versions? 

The travis build is failing because there are still some tests in there call `expectExceptionMessageRegExp`.

Reply:
That's what I did here https://github.com/mockery/mockery/pull/1049/files#diff-491577b1fec453820a42849002bd0828R41

Reply:
@davedevelopment Yes there are issues with signatures. I'll try to fix travis build so it passes.